### ISSUE TITLE
works with Win10 WSL Debian, but doesn't check for Java

### ISSUE THREAD

**Issue by markstinson** [kicker22004/BSCC-MC-Edition#6]
I just finished running BSCC in Windows 10 using Windows Subsystem for Linux ( WSL Debian, Mojang Vanilla 1.13.2, openjdk-8-jre-headless, BSCC set Max memory to 1024M, Min memory 512M) and connecting using the Java Client. On my Ryzen 5 2600 w/ 8GB & GTX750, it ran smooth as silk for a single player.

However, when issuing `BSCC start` it originally acted like it was going to start but there was no process. BSCC didn't throw an error that java was not in the path and did not offer installing "openjdk-8-jre-headless" given it was missing.

Sure, it could be assumed that Java should be installed, but given the install script does have mention of Oracle Java (due to RasPi), it may be worthwhile to toss the lack of java error (and offer to install openjdk-8-jre-headless if available)

**Comment by kicker22004** [comment]
Yes i'm working on this now, and i'll leave the installation of java up to the users as Oracle keeps locking downloads with ease. I'll have the installer check for java -v to return an actual version number and if not error out. Sorry I didn't get these sooner.
